app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -51,22 +52,26 @@ class BindingTariffClassificationConnector @Inject()(appConfig: AppConfig, clien
     client.GET[Option[Case]](url)
   }
 
+  private def buildQueryUrl(types : Seq[ApplicationType] = Seq(ApplicationType.BTI,ApplicationType.LIABILITY_ORDER), withStatuses: String,
```

Comment:
Is there any value in just building a Search object and calling the search method below instead?

Reply:
Possibly could be. I'll have a look at that while implementing the extension to this for DIT-1113, as will need to send some different queries then.